src/realm/object-store/sync/app.cpp

```diff
@@ -281,6 +298,7 @@ void App::update_hostname(const util::Optional<SyncAppMetadata>& metadata)
 void App::update_hostname(const std::string& hostname, const Optional<std::string>& ws_hostname)
 {
     // Update url components based on new hostname value
+    log("App: update_hostname: %1 | %2", hostname, ws_hostname);
```

Comment:
were all these tests failing or something?

Reply:
Nothing was failing - just to show the hostnames being configured, especially if it changes as a result of a redirection.